assets/roles_controller.yaml

```diff
@@ -0,0 +1,30 @@
+kind: Role
```

Comment:
these files are IMO named backward. This should be called `controller_rbac_roles.yaml` and other file should be called `controller_namespace.yaml` etc

Reply:
I've updated all of these except for the deployment. I'd like to keep the deployment this way so that it's clearly different from the setup yamls.